ecosystem/sep-0001.md

```diff
@@ -42,7 +42,7 @@ content-type: text/plain
 
 `stellar.toml` is in [TOML](https://github.com/toml-lang/toml) file format. The `stellar.toml` fields and sections are described below. You should complete as much of this as you can. Many Stellar apps, including important wallets and exchanges, make decisions on which tokens to support based on the completeness of their Account Information and Documentation sections.
 
-### Account Information
+### Organization Information
```

Comment:
Nothing in this section looks like it relates to an organization to me. It's all server URLs, and accounts lists. Maybe we should call this "General Information"?

Reply:
@leighmcculloch you're a modern day poet.